src/operators/localoperator.jl

```diff
@@ -96,6 +96,15 @@ function Base.repeat(O::LocalOperator, m::Int, n::Int)
     return LocalOperator(lattice, terms...)
 end
 
+# Real and imaginary part
+# -----------------------
+function Base.real(O::LocalOperator)
+    return LocalOperator(O.lattice, Tuple(ind => real(op) for (ind, op) in O.terms)...)
```

Comment:
```suggestion
    return LocalOperator(O.lattice, (ind => real(op) for (ind, op) in O.terms)...)
```
I don't think you need to first construct a tuple for this to work, it should be possible to directly splat the generator

Reply:
Just fixed (and for `imag` as well). Thanks!